include/openenclave/bits/sgx/seal.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_BITS_SGX_SEAL_H
+#define _OE_BITS_SGX_SEAL_H
+
+#include <openenclave/bits/sgx/sgxtypes.h>
+
+OE_EXTERNC_BEGIN
+
+/**
+ * SGX specific seal settings
+ */
+typedef enum _oe_seal_setting_name_sgx
+{
+    OE_SEAL_SETTING_SGX_KEYNAME = 0x100,
+    OE_SEAL_SETTING_SGX_ISVSVN,
+    OE_SEAL_SETTING_SGX_CET_ATTRIBUTES_MASK,
+    OE_SEAL_SETTING_SGX_CPUSVN,
+    OE_SEAL_SETTING_SGX_ATTREMASK,
+    OE_SEAL_SETTING_SGX_XFRMMASK,
+    OE_SEAL_SETTING_SGX_MISCMASK,
+    OE_SEAL_SETTING_SGX_CONFIGSVN,
+    OE_SEAL_SETTING_SGX_MAX
```

Comment:
Is this "MAX" used somewhere?  If not, remove it.

Reply:
I was thinking of doing `OE_SEAL_SETTING_SGX_KEYNAME = OE_SEAL_SETTING_TEE_AGNOSTIC_MAX` here but somehow forgot to do that (and used `0x100` instead). How about we do the following?

```c
enum
{
    OE_SEAL_SETTING_SGX_KEYNAME = OE_SEAL_SETTING_TEE_AGNOSTIC_MAX,
    OE_SEAL_SETTING_SGX_ISVSVN,
    OE_SEAL_SETTING_SGX_CET_ATTRIBUTES_MASK,
    OE_SEAL_SETTING_SGX_CPUSVN,
    OE_SEAL_SETTING_SGX_ATTREMASK,
    OE_SEAL_SETTING_SGX_XFRMMASK,
    OE_SEAL_SETTING_SGX_MISCMASK,
    OE_SEAL_SETTING_SGX_CONFIGSVN,
    // OE_SEAL_SETTING_SGX_MAX removed
};
```